app/styles/mobile.scss

```diff
@@ -14,69 +18,85 @@
 
 
   .clearfix {
-    padding-top:120px;
+    padding-top: 120px;
   }
 
-  html {
-    text-align: center;
-    th {
+  .custom {
+    padding-bottom: 10px;
       text-align: center;
+      border: none;
+  }
+
+  #footer {
+    background-color: $black;
+    text-align: center;
+    a:hover {
+      color: $purple;
+      color: $white;
+      cursor: pointer;
+    }
+    center {
+      color:$white;
     }
   }
 
-  .navbar-default {
-    height: 57px;
-    background-color: black;
-    border-color: black;
+  #form-spacing {
+    width: 100%;
+    text-align: center;
+  }
 
+  #form-spacing2 {
+    width: 100%;
   }
 
+  html {
+    text-align: center;
+    th {
+      text-align: center;
+    }
+  }
 
   #logo {
-    width: 90px;
+    bottom: 15px;
     height: 54px;
     position: relative;
-    bottom: 15px;
+    width: 90px;
   }
 
-  #footer {
+  #main_content {
```

Comment:
Avoid using id selectors<br>Selector `main_content` should be written in lowercase with hyphens